$NetBSD: patch-scripts_libbrotlidec.pc.in,v 1.1 2021/03/21 19:24:24 nros Exp $

Remove wrong rpath argument on Linux and use PKGCONFIG_OVERIDE instead

--- scripts/libbrotlidec.pc.in.orig	2021-03-21 12:52:03.713184864 +0000
+++ scripts/libbrotlidec.pc.in
@@ -7,6 +7,6 @@ Name: libbrotlidec
 URL: https://github.com/google/brotli
 Description: Brotli decoder library
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -R${libdir} -lbrotlidec
+Libs: -L${libdir} -lbrotlidec
 Requires.private: libbrotlicommon >= 1.0.2
 Cflags: -I${includedir}
